### PR TITLE
CSV Cleanups and additions

### PR DESCRIPTION
**Pull request by VirusShell**
I think I did this right. Just updated it to the main branch. Added a handful of new entries to the lists, removed any spelling errors I could find, removed all of the duped entries for all the csv lists as well.

Started going thru the templates.csv, as there are many small little issues throughout it, but things are easy to break because of how bracketing was done in it. Holding onto it for later.

Planning to go thru the scripts for the few spelling errors I've found as well, and probably reword a few things for grammar.

### CODE REVIEW

**Comment by VirusShell** [comment]
hmmmm... did it update add 5 updated files?

File: csvfiles/devmessages.csv [hunk 1]
```diff
@@ -50,11 +50,15 @@ Prompt evaluation failed. restarting prompt generation...
...
 Prompt illogical. restarting prompt generation...
 But what about two button prompt...
 Changing 1 to 0, and 0 to 1...
 The next one will be better...
-Summoning the techno-wizards...
\ No newline at end of file
+Summoning the techno-wizards...
+Great Scott! The Flux Capacitor needs more Plutonium
+We require more vespene gas
+No regerts!!!
```
Comment (AIrjen) [inline csvfiles/devmessages.csv:63]: Love it!

File: csvfiles/locations.csv [hunk 2]
```diff
@@ -675,8 +675,9 @@ Wormhole
...
 Labyrinth
-color field
+-color- field
 alternate dimensions
 jupiter clouds
 crack in the earth
-Mushroom Kingdom
\ No newline at end of file
+Mushroom Kingdom
+-animal- den
```
Comment (AIrjen) [inline csvfiles/locations.csv:683]: Oh I like this one!

File: csvfiles/special_lists/buildoutfit.csv [hunk 3]
```diff
@@ -8,8 +9,10 @@
 -material- -color- -outfit-
 -descriptor- -color- -outfit-
 -descriptor- -material- -outfit-
+-descriptor- -material- -color- -outfit-
 -job- -outfit-
 -rpgclass- -minioutfit-
 -season- -minioutfit-
+-animal--style -outfit-
```
Comment (AIrjen) [inline csvfiles/special_lists/buildoutfit.csv:16]: Oh, I thought this was a typo. But this one is very smart. Like it!

**Comment by AIrjen** [comment]
Amazing work! I only made one change. I removed the "space" objects from the objects list. They are in the "space" list, which I moved over only recently. So -objecttotal- will also get the space objects. 

If you have other suggestions or changes, this is perfect way to do it.

**Comment by VirusShell** [comment]
Awesome to hear that! Well I'll keep at it then.

Currently looking at the time periods to make those work with a couple of prompt I was testing. Managed to get the list to generate correctly, so now I'm trying to decide on a better way to reword things in the list for it to work better. It's hit or miss, even while regular testing. But I figure it's something that works well enough in the quick additions or for the locations

**Pull request merged**